lib/t_web/channels/feed_channel.ex

```diff
@@ -251,18 +212,6 @@ defmodule TWeb.FeedChannel do
           screen_width: screen_width,
           expiration_date: expiration_date
         })
-
```

Comment:
Since there is no need for multiple function clauses, I'd move match struct pattern matching inside the function body, it would make it a bit easier to read.

Reply:
```elixir
Enum.map(matches, fn match ->
  %Matches.Match{...} = match
  render_match(%{...})
end)
```